DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,16 @@ public function getConfigTreeBuilder()
                     ->defaultValue(array())
                     ->prototype('scalar')->end()
                 ->end()
+                ->arrayNode('use_stacktrace_js')
+                    ->info('add StackTrace.js as logging provider')
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('path')
```

Comment:
what path should this be, please add a clarification as "->info(..." to this node and if possible a default value => e.g. an absolute url to CDN where stacktrace.js can be found